recipe/meta.yaml

```diff
@@ -28,12 +28,10 @@ requirements:
     - cf_units
     - dask >=0.17.1
     - matplotlib
-    - netcdf4 <1.4
     - numpy
     - pyke
     - scipy
     - six
-    - iris-grib  # [not (win or py3k)]
```

Comment:
I actually love to see this go away but when I removed it in the past I believe someone from SciTools requested me to put it back.